Mobile/ShopLens/MigrationsConsole/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using ShopLensWeb;
+
+namespace MigrationsConsole
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var useLocalDb = Convert.ToBoolean(ConfigurationManager.AppSettings["UseLocalDb"]); 
+            var connectionString = ConfigurationManager.ConnectionStrings["DatabaseSource"].ConnectionString;
+            
+            using (var dbContext = new ShopLensContext(connectionString, useLocalDb))
+            {
+                dbContext.Database.Migrate();
+
+                //DeleteAllUsers(dbContext);
+                //AddTestUsers(dbContext);
+                ListAllUsers(dbContext);
```

Comment:
There should really be a Console.Readline() or Console.ReadKey() here, I mean the console just disappears when I build the project.

Reply:
I use a terminal for example, and a readline/readkey would be excessive for me. As this is a kind of "test" project, with test methods just to show how things are done, I think everyone should just configure it as they like, with or without readlines.

Unless this project should become more "official" migration tool, then we have to clean it up (like remove those test methods for example).